### DISCUSSION TITLE
Support for composite id in MockExecResult

### DISCUSSION THREAD

**Discussion by msoral-lidya** [SeaQL/sea-orm#2261] [Q&A]
Hello, new user here. I am trying to test inserting row(s) into a table that has composite id. 

From the [documentation](https://www.sea-ql.org/SeaORM/docs/basic-crud/insert/#insert-one):
> Insert an active model and get back the last insert id. Its type matches the model's primary key type, so it could be a tuple if the model has a composite primary key.

Looking at the [documentation about mocking execution results](https://www.sea-ql.org/SeaORM/docs/write-test/mock/#mocking-execution-result) I understand that I need to append `MockExecResult`s into the `MockDatabase`. However, I could not find any way to achieve this with composite ids.

Looking at the implementation I don't think it's supported. Please let me know if I am missing something.

Thanks in advance.

**Comment by xinyifly** [comment]
I currently use UUID generated in application for primary key. Besides type u64 mismatch, there isn't a way to pre-know application generated UUID in tests without looking at insert statement.

Update: I found that I can use `append_query_results` instead of `append_exec_results` for insert.
